docs/resources/samples/convert-csv.md

```diff
@@ -102,45 +107,57 @@ function main(workbook: ExcelScript.Workbook, csv: string) {
+### Script testing
+
+To test the script without using Power Automate, assign a value to `csv` before using it. Try adding the following code as the first line of the `main` function and pressing **Run**.
 
 ```TypeScript
-function main(workbook: ExcelScript.Workbook, csv: string) {
-  let sheet = workbook.getWorksheet("Sheet1");
+  csv = `1, 2, 3
+         4, 5, 6
+         7, 8, 9`;
+```
 
-  /* Convert the CSV data into a 2D array. */
-  // Trim the trailing new line.
-  csv = csv.trim();
+### Semicolon-separated files and other alternative separators
 
-  // Split each line into a row.
-  let rows = csv.split("\r\n");
-  rows.forEach((value, index) => {
-    /*
-     * For each row, match the comma-separated sections.
-     * For more information on how to use regular expressions to parse CSV files,
-     * see this Stack Overflow post: https://stackoverflow.com/a/48806378/9227753
-     */
-    let row = value.match(/(?:,|\n|^)("(?:(?:"")*[^"]*)*"|[^",\n]*|(?:\n|$))/g);
-
-    // Check for blanks at the start of the row.
-    if (row[0].charAt(0) === ',') {
-      row.unshift("");
-    }
+Some regions use semicolons (';') to separate cell values instead of commas. In this case, you need to change the following lines in the script.
 
-    // Remove the preceding comma.
-    row.forEach((cell, index) => {
-      row[index] = cell.indexOf(",") === 0 ? cell.substr(1) : cell;
-    });
+1. Replace the commas with semicolons in the regular expression statement. This starts with `let row = value.match`.
 
-    // Create a 2D-array with one row.
-    let data: string[][] = [];
-    data.push(row);
+    ```TypeScript
+    let row = value.match(/(?:;|\n|^)("(?:(?:"")*[^"]*)*"|[^";\n]*|(?:\n|$))/g);
+    ```
 
-    // Put the data in the worksheet.
-    let range = sheet.getRangeByIndexes(index, 0, 1, data[0].length);
-    range.setValues(data);
-  });
+1. Replace the comma with a semicolon at in the check for the blank first cell. This starts with `if (row[0].charAt(0)`.
 
-  // Add any formatting or table creation that you want.
-}
-```
+    ```TypeScript
+    if (row[0].charAt(0) === ';') {
+    ```
+
+1. Replace the comma with a semicolon in the line that removes the separation character from the displayed text. This starts with `row[index] = cell.indexOf`.
+
+   ```TypeScript
+      row[index] = cell.indexOf(";") === 0 ? cell.substr(1) : cell;
+    ```
+
+If your file uses tabs or any other character to separate the values, replace the `;` in the above substitutions with `\t` or whatever character is being used.
```

Comment:
While reading this section on the staging site, I feel inclined to make this last sentence a !NOTE. Right now, it's short and feels like an afterthought, but it's a really useful bit of information. The formatting of a !NOTE would help highlight it.